hypothesis-python/tests/cover/test_reflection.py

```diff
@@ -59,16 +59,6 @@ def foo(a, b, c):
     do_conversion_test(foo, (1,), {"c": 2, "b": "foo"})
 
 
-def test_populates_defaults():
-    def bar(x=[], y=1):
-        pass
-
-    assert convert_keyword_arguments(bar, (), {}) == (([], 1), {})
```

Comment:
A surprisingly complementary case to the above: previously this would populate defaults for pos-or-keyword arguments, but not for keyword-only.  As above, we resolved the inconsistency in favor of passing fewer arguments.